packages/flutter_tools/lib/src/commands/build_ios_framework.dart

```diff
@@ -525,7 +525,15 @@ end
       '-create-xcframework',
       for (Directory framework in frameworks) ...<String>[
         '-framework',
-        framework.path
+        framework.path,
+        ...framework.parent
+            .listSync()
+            .where((FileSystemEntity entity) =>
+                entity.basename.endsWith('bcsymbolmap') ||
+                entity.basename.endsWith('dSYM'))
+            .map((FileSystemEntity entity) =>
+                <String>['-debug-symbols', entity.path])
+            .expand((i) => i)
```

Comment:
```suggestion
            .expand<String>((List<String> parameter) => parameter)
```
You have an analyzer warning here:
```
 info • Specify type annotations • packages/flutter_tools/lib/src/commands/build_ios_framework.dart:536:22 • always_specify_types
```